contrib/kubespray/roles/requirement/devbox/tasks/ubuntu.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: "[1/2] Check if docker is installed on your dev-box or not."
+  raw: command -v docker
+  register: command_docker
+  failed_when: false
+  changed_when: false
+  check_mode: false
+  environment: {}
+
+- name: "[2/2] Check if docker is installed on your dev-box or not."
+  fail:
+    msg: "Unable to find docker in your dev-box machine"
+  changed_when: false
+  check_mode: false
+  environment: {}
+  when:
+    - command_docker.rc != 0
+
```

Comment:
could you also check docker version?
```sh
docker info -f '{{json .ServerVersion}}'
```

Reply:
Please provide the lowest version

Reply:
Old docker version in the host will be removed: 
https://github.com/kubernetes-sigs/kubespray/blob/master/roles/container-engine/docker/tasks/pre-upgrade.yml#L2
https://github.com/kubernetes-sigs/kubespray/blob/master/roles/container-engine/docker/defaults/main.yml#L67

Reply:
it's dev box, not in cluster

Reply:
The version specifed in environment will be installed:
https://github.com/kubernetes-sigs/kubespray/blob/master/roles/container-engine/docker/vars/debian.yml#L18
https://github.com/kubernetes-sigs/kubespray/blob/master/roles/container-engine/docker/tasks/main.yml#L151

Reply:
Does our dev-box have some limitation of docker version?

Reply:
not sure, but at least [< 1.10 may not pull images from new registry](https://docs.docker.com/engine/breaking_changes/)

Reply:
Done. If version <= 1.10, will output failure.